Gordon360/Controllers/RequestsController.cs

```diff
@@ -117,15 +112,15 @@ public ActionResult<REQUEST> Post([FromBody] REQUEST membershipRequest)
         /// <summary>
         /// Updates a membership request
         /// </summary>
-        /// <param name="id">The membership request id</param>
+        /// <param name="membershipRequestID">The membership request id</param>
         /// <param name="membershipRequest">The updated membership request object</param>
         /// <returns>The updated request if successful. HTTP error message if not.</returns>
         [HttpPut]
-        [Route("{id}")]
+        [Route("{membershipRequestID}")]
         [StateYourBusiness(operation = Operation.UPDATE, resource = Resource.MEMBERSHIP_REQUEST)]
```

Comment:
I think that this will only authorize SiteAdmins because UDPATE for MEMBERSHIP_REQUEST expects a `status` argument as well as the `membershipRequestID` parameter.

Reply:
I am removing status from StateYourBusincess since it is unused and parameter validation should happen in the service/controller rather than the authorization logic.